README.md

```diff
@@ -11,4 +11,28 @@ Code Development Note
 
 Some of the code in the package deblur has been derived from [QIIME](http://qiime.org).
 The contributors to these specific QIIME modules have granted permission
-for this porting to take place and put under the BSD license.
\ No newline at end of file
+for this porting to take place and put under the BSD license.
+
+Install
+=======
+
+```
+python setup.py install
+```
+
+
+Example usage
+=============
+
+The input to deblur workflow is a demultiplexed FASTA file. The deblur algorithm is
+designed to work on individual samples.
+
+```
+deblur workflow all_samples.fna db.fna output.biom
+```
+
+or
+
+```
+parallel_deblur workflow all_samples.fna db.fna output.biom
```

Comment:
Will review in a hour or so
On Mar 9, 2016 6:05 AM, "Evguenia Kopylova" <notifications@github.com>
wrote:

> In README.md
> <https://github.com/biocore/deblur/pull/30#discussion_r55521545>:
>
> > +
> > +
> > +Example usage
> > +=============
> > +
> > +The input to deblur workflow is a demultiplexed FASTA file. The deblur algorithm is
> > +designed to work on individual samples.
> > +
> > +```
> > +deblur workflow all_samples.fna db.fna output.biom
> > +```
> > +
> > +or
> > +
> > +```
> > +parallel_deblur workflow all_samples.fna db.fna output.biom
>
> to remove for now ..
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/deblur/pull/30/files#r55521545>.
>